app/controllers/order/goods_details.js

```diff
@@ -11,6 +11,10 @@ export default Ember.Controller.extend({
   otherDetails: "",
   sortProperties: ["id"],
   sortedGcRequests: Ember.computed.sort("model.goodcityRequests", "sortProperties"),
+  gcRequestNull: Ember.computed("model.goodcityRequests", function() {
+                    return (this.get('model.goodcityRequests').length === 0);
```

Comment:
No need to compare with zero.
!this.get('model.goodcityRequests') should work

Reply:
done, thanks :+1: